dashboard/app/views/layouts/_header.html.haml

```diff
@@ -9,7 +9,7 @@
 
   user_type = nil
 
-  header_class = 'header2-wrapper'
```

Comment:
How bad does this page look if you revert everything except for this line? This is what users may see for up to 12 hours after we ship this change, even if we flush the cloudfront cache. This is based on the following cache headers for style.css: `public, max-age=43200, s-maxage=21600`

Reply:
assuming we are referring to this page as /style.css. If we're using some kind of digested version, then it would be good to confirm this page actually uses a digested URL instead.

Reply:
Good call.  I will update this PR to include the same CSS for both header* and header2* classes, and not yet flip to use the new one.  Once we know everybody has new CSS, then we can flip to use the new one (and probably get rid of the old one at that point too).

Reply:
Actually, that's just for pegasus.  Dashboard digests its CSS properly so we should be able to change its CSS & HTML now just fine.

Reply:
Addendum to two comments ago, we should then wait until all web caches have the new HTML before removing header2* from the CSS.  So this is a three phase change, as you suggested in conversation @davidsbailey.

Reply:
@davidsbailey Description updated and second commit made to address this issue.